adoc/chapters/programming_interface.adoc

```diff
@@ -5540,358 +5423,318 @@ queue, the sub-buffer's alignment must be compatible with both the primary
 ----
-   a@ Returns the allocator provided to the buffer.
+_Returns:_ The allocator provided to the buffer.
+'''
 
-a@
-[source]
+.[apititle]#buffer::get_access#
+[source,role=synopsis,id=api:buffer-get-access]
 ----
 template <access_mode Mode = access_mode::read_write,
           target Targ = target::device>
-accessor<T, Dimensions, Mode, Targ> get_access(handler& commandGroupHandler)
-----
-   a@ Returns a valid [code]#accessor# to the buffer with the specified
-      access mode and target in the command group buffer.
-      The value of target can be [code]#target::device#,
-      [code]#target::constant_buffer# or
-      [code]#target::host_task#.
-
-a@
-[source]
-----
-template <access_mode Mode>
-accessor<T, Dimensions, Mode, target::host_buffer> get_access()
-----
-   a@ Deprecated in SYCL 2020.  Use [code]#get_host_access()# instead.
+accessor<T, Dimensions, Mode, Targ> get_access(handler& commandGroupHandler); (1)
 
-Returns a valid host [code]#accessor# to the buffer with the specified
-access mode and target.
-
-a@
-[source]
-----
 template <access_mode Mode = access_mode::read_write,
           target Targ = target::device>
-accessor<T, Dimensions, Mode, Targ> get_access(handler& commandGroupHandler,
-                                               range<Dimensions> accessRange,
-                                               id<Dimensions> accessOffset = {})
+accessor<T, Dimensions, Mode, Targ>
+get_access(handler& commandGroupHandler, range<Dimensions> accessRange,
+           id<Dimensions> accessOffset = {}); (2)
+
+template <typename... Ts> auto get_access(Ts...); (3)
+
+template <typename... Ts> auto get_host_access(Ts...); (4)
 ----
-   a@ Returns a valid [code]#accessor# to the buffer with the specified access
-      mode and target in the command group buffer.  The accessor is a
-      <<ranged-accessor>>, where the range starts at the given offset from the
-      beginning of the buffer.  The value of target can be
-      [code]#target::device#, [code]#target::constant_buffer# or
-      [code]#target::host_task#.
+_Returns (1):_ A valid [code]#accessor# to the buffer with the specified access
+mode and target in the command group buffer. The value of target can be
+[code]#target::device#, [code]#target::constant_buffer# or
+[code]#target::host_task#.
 
-Throws an [code]#exception# with the [code]#errc::invalid# error code if
+_Returns (2):_ A valid [code]#accessor# to the buffer with the specified access
+mode and target in the command group buffer.  The accessor is a
+<<ranged-accessor>>, where the range starts at the given offset from the
+beginning of the buffer.  The value of target can be [code]#target::device#,
+[code]#target::constant_buffer# or [code]#target::host_task#.
+
+_Returns (3):_ A  valid [code]#accessor# as if constructed via passing the
+buffer and all provided arguments to the [code]#accessor# constructor.
+
+Possible implementation:
+
+[code]#+return accessor{*this, args...};+#
```

Comment:
```suggestion
```

I don't think we need possible implementations in the spec.

Reply:
Good question. I think I agree, but I didn't want to change the current descriptions too much. I'll happily remove it if people agree though.